internal/server/server_test.go

```
@@ -0,0 +1,1000 @@
+	}
+	if err := os.WriteFile(sessionPath, data, 0o644); err != nil {
+		t.Fatalf("write legacy session: %v", err)
+	}
+
+	dbPath := filepath.Join(dataDir, "inbox.db")
+	raw, err := sql.Open("sqlite", dbPath)
+	if err != nil {
+		t.Fatalf("open legacy database: %v", err)
+	}
+	defer raw.Close()
+	if _, err := raw.Exec(`
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)